src/panels/config/entities/ha-config-entities.ts

```diff
@@ -280,6 +288,10 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
                       ? this.hass.localize(
                           "ui.panel.config.entities.picker.status.disabled"
                         )
+                      : entity.hidden_by
+                      ? this.hass.localize(
+                          "ui.panel.config.entities.picker.status.hidden"
```

Comment:
This one doesn't work

![image](https://user-images.githubusercontent.com/1444314/158697476-82471d14-4410-4f08-83cc-c7f5b5267fd0.png)

Reply:
oh whoops